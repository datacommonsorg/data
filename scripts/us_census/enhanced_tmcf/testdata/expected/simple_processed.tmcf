Node: E:simple_processed->E0
typeOf: schema:Place
dcid: C:simple_processed->GEO_ID

Node: E:simple_processed->E1
typeOf: schema:StatVarObservation
value: C:simple_processed->B01001_001E
variableMeasured: T:Estimate!!Total
observationAbout: E:simple_processed->E0
observationDate: "2020"
