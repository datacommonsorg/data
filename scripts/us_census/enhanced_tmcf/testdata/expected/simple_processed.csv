GEO_ID,B01001_001E
country/USA,25417
country/USA,17595
country/USA,598065
